src/main/java/dk/kb/util/webservice/Service2ServiceRequest.java

```diff
@@ -84,43 +93,6 @@ public static <T> T httpCallWithOAuthToken (URI uri , String httpMethod, T objec
 
     }
 
-    /** 
-     * <p>
-     * Make service call to another web service and set the same OAuth token on the call that was used for the initiating service call. 
-     * <p>
-     * 
-     * Establish a connection to the given {@code uri}, extract all headers and construct a
-     * {@link ContinuationInputStream} with the headers and response stream from the {@code uri}.
-     * 
-     * @param uri full URI for a call to a webservice.
-     * @param tokenMapper maps the String header {@link ContinuationUtil#HEADER_PAGING_CONTINUATION_TOKEN}
-     *                    to the concrete token type.
-     * @param requestHeaders optional headers for the connection. Can be null.
-     * @return an {@code InputStream} with the response.
-     * @throws IOException if the connection failed.
-     */
-    public static <C2> ContinuationInputStream<C2> continuationInputStreamFromWithOAUthToken(
```

Comment:
Why are we deleting this method? Not needed anyways?

Reply:
Yes, moved to  ContinuationInputStream. Did not work as a util method in all cases. <DTO, LONG>